### ISSUE TITLE
RuntimeWarning: invalid value encountered in sqrt

### ISSUE THREAD

**Issue by pachacamac** [wannesm/dtaidistance#122]
getting the following error when I try to execute one of the examples:

```python3.8/site-packages/dtaidistance/dtw.py:392: RuntimeWarning: invalid value encountered in sqrt
  dtw = np.sqrt(dtw)
```

the example code from the readme I tried was:

```from dtaidistance import dtw
from dtaidistance import dtw_visualisation as dtwvis
import random
import numpy as np
x = np.arange(0, 20, .5)
s1 = np.sin(x)
s2 = np.sin(x - 1)
random.seed(1)
for idx in range(len(s2)):
    if random.random() < 0.05:
        s2[idx] += (random.random() - 0.5) / 2
d, paths = dtw.warping_paths(s1, s2, window=25, psi=2)
best_path = dtw.best_path(paths)
dtwvis.plot_warpingpaths(s1, s2, paths, best_path)
```

**Comment by wannesm** [comment]
Thanks for the report and reproducible example. This is related to #114 . A fix is in the works.

**Comment by wannesm** [comment]
This is fixed in b2d79c12ad7fe416521790453198ec714da438f7.
